cloudify_agent/api/pm/base.py

```diff
@@ -298,6 +304,8 @@ def create_celery_conf(self):
             'broker_vhost': self.broker_vhost,
             'cluster': self.cluster
         }
+        if self.broker_heartbeat:
+            config['broker_heartbeat'] = self.broker_heartbeat
```

Comment:
This is the line which adds the setting to the `broker_config.json` file.
The other changes are about getting the value to this place - starting from agent config, via the daemons command.